### PR TITLE
Upgrade to PHP-CS-Fixer 3.x

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log

- Upgrading to the `PHP-CS-Fixer 3.x`.
- Adding the `composer run php-cs-fixer` command in the GitHub actions running.

### CODE REVIEW

File: tests/Strategy/JsonStrategyTest.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ public function testTo()
             $text = "\xB1\x31";
             $jsonStrategy->render($text);
             $this->assertInstanceOf(
-                ResponseInterface ::class,
+                ResponseInterface::class,
```
Comment (peter279k) [inline tests/Strategy/JsonStrategyTest.php:93]: These three lines are fixed via running the `composer run php-cs-fixer` command.

File: php_cs.dist [hunk 2]
```diff
@@ -23,7 +23,7 @@ $finder = PhpCsFixer\Finder::create()
     ->exclude('vendor')
     ->in(__DIR__);
 
-return PhpCsFixer\Config::create()
+return (new PhpCsFixer\Config())
```
Comment (peter279k) [inline php_cs.dist:26]: To be compatible with `PHP-CS-Fixer 3.x`, it should use the `(new PhpCsFixer\Config())` instead.

File: php_cs.dist [hunk 3]
```diff
@@ -50,7 +50,7 @@ return PhpCsFixer\Config::create()
             'order' => ['use_trait']
         ],
         'ordered_imports' => [
-            'importsOrder' => [
+            'imports_order' => [
```
Comment (peter279k) [inline php_cs.dist:53]: The rule name changes into `imports_order` since the `PHP-CS-Fixer 3.x` version.

**Review by arcostasi — APPROVED** [review]

**Comment by arcostasi** [comment]
Thank you very much @peter279k

Passed all tests!
test-ok

**Pull request merged**